README.md

```diff
@@ -0,0 +1,99 @@
+[![Latest Release](https://img.shields.io/github/release/AmericanMedicalAssociation/AMA-style-guide.svg)](https://github.com/AmericanMedicalAssociation/AMA-style-guide/releases/latest)
```

Comment:
I need to go through this and take out the parts that aren't yet relevant to SG2.

Reply:
@designerneil I tweaked some wording and stuff and checked to make sure the stuff in readme.md makes sense in SG2-land. It looks like you'd already done a lot of that too, so thanks! Do you mind giving my changes a once-over? If everything looks good to you, I think readme.md will be good to go for the time being.